### PR TITLE
Turn off throttle option when requesting for an availability tile from the parent terrain layer

### PR DESCRIPTION
**Pull request by baothientran**
The function `checkLayer()` in the `CesiumTerrainProvider.js` file [is responsible for requesting an availability tile for the parent layer](https://github.com/CesiumGS/cesium/blob/13b40ad846d70f58b0385d0b69a9316e4b599706/Source/Core/CesiumTerrainProvider.js#L1280). However, the promise is delayed for a long time or never get resolved especially where the tile is overlapped with the child layer, which makes `QuadtreePrimitive` upsamples the tile from the parent tile. Turning off the throttle option in the Request object does solves the issue though. This is consistent with other requests coming from the other terrain code

This is the parent layer when all the tiles are available. Notice the three lines on the head of the two tiles at level 10
Screenshot_20200818_170231

This is the missing tiles from the parent layer when delay happens. Notice there are no three lines on the head of the tiles at level 10. Keeping zooming in doesn't solve the problem
Screenshot_20200818_170108

@lilleyse Can you please take a look at it? Please let me know if I need to add anything

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @baothientran!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/terrain-throttle/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by lilleyse** [comment]
@baothientran Definitely 👍 on the change. The terrain and imagery system used to throttle requests in the `RequestScheduler` but it clashed  with 3D Tiles' priority system so we reverted it and let terrain prioritize/throttle according to its own rules. I think this code just never got reverted.

Can you make the same change for other terrain/imagery providers? A search for `throttle: true` brings up a few more results. Ultimately `throttle: true` should be removed everywhere except in `Cesium3DTile`, `ResourceSpec`, and `RequestSchedulerSpec`.

**Comment by baothientran** [comment]
@lilleyse I turned off the throttle options for all the terrain and imagery provider. Some tests are fixed along the way, but it shouldn't change the testing cases' purposes

**Comment by lilleyse** [comment]
@baothientran could you the fix to `CHANGES.md`?

**Comment by lilleyse** [comment]
@kring can you confirm that this is a good fix?

File: Specs/Core/ArcGISTiledElevationTerrainProviderSpec.js [hunk 1]
```diff
@@ -474,12 +474,11 @@ describe("Core/ArcGISTiledElevationTerrainProvider", function () {
...
-            0,
-            0,
-            0,
-            createRequest()
-          );
+          var request = new Request({
+            throttle: true,
+            throttleByServer: true,
+          });
+          promise = terrainProvider.requestTileGeometry(0, 0, 0, request);
```
Comment (kring) [inline Specs/Core/ArcGISTiledElevationTerrainProviderSpec.js:481]: Isn't it a problem that we're no longer calling `createRequest` here? So the request has no URL, for one thing.

Comment (baothientran) [reply Specs/Core/ArcGISTiledElevationTerrainProviderSpec.js:481]: it seems like createRequest() returns an empty request object with only `throttleByServer` option on. I kind of create the new Request for only the test case. Should I modify the createRequest() with throttle to be true for all of them?

Comment (kring) [reply Specs/Core/ArcGISTiledElevationTerrainProviderSpec.js:481]: Oh ok, I hadn't realized `createRequest` also created a request with a URL. That makes sense, no need to change anything.

**Pull request merged**